src/App.java

```diff
@@ -0,0 +1,283 @@
+                                System.out.println("Call method readFile from List<Task> " +
+                                        "tasks = new ArrayList<>()");
+                                System.out.println("Call method deletedTaskById from List<Task> " +
+                                        "tasks = new ArrayList<>()");
+                                break;
+                            }
+                            case 2: {
+                                String[] byPeriodMenu = new String[]{"BY PERIOD"};
+                                outputMenu(byPeriodMenu);
+                                System.out.println("Call method readFile from List<Task> " +
+                                        "tasks = new ArrayList<>()");
+                                System.out.println("Call method deletedTaskByPeriod from List<Task> " +
+                                        "tasks = new ArrayList<>()");
+                                break;
+                            }
+                            case 0: {
+                                userChoice = 0;
+                                break;
+                            }
+                        }
+                    }
+                    userChoice = -1;
+                    break;
+                }
+                default: {
+                    System.out.println("Please make your choice");
+                    break;
+                }
+                case 0: {
+                    userChoice = 0;
+                    break;
+                }
+            }
+        }
+    }
+
+    // Method of validation maximum possible choice
+    public static int getUserIntChoice(int maxChoice) {
+        Scanner scanner = new Scanner(System.in);
+        int userChoice = 0;
+        boolean isInputNumber;
+        do {
+            isInputNumber = scanner.hasNextInt();
+            if (!isInputNumber) {
+                System.out.println("Please enter a number");
+                scanner.next();
+            } else {
+                userChoice = scanner.nextInt();
+                if (userChoice > maxChoice) {
+                    System.out.println("You entered an incorrect number");
+                } else {
+                    break;
+                }
+            }
+        }
+        while (!isInputNumber || userChoice > maxChoice);
+        return userChoice;
+    }
+
+    //ToDo to add method to maximumPossibleChoice
+    private static String getStringNotEmpty() {
+        Scanner scanner = new Scanner(System.in);
+        String inputString = scanner.nextLine();
+        boolean isStringEmpty;
+        do {
+            isStringEmpty = inputString.isEmpty();
+            if (isStringEmpty) {
+                System.out.println("String cannot be Empty.Please string!\n");
+                inputString = scanner.nextLine();
+            } else {
+                break;
+            }
+        }
+        while (isStringEmpty);
+        return inputString;
+    }
```

Comment:
remove getStringNotEmpty ()

and create String getUserInput() without validation and sout message